src/sparql/qc/mondo/qc-animal-disease-rare.sparql

```diff
@@ -0,0 +1,25 @@
+prefix IAO: <http://purl.obolibrary.org/obo/IAO_>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+prefix def: <http://purl.obolibrary.org/obo/IAO_0000115>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix mondoSparqlQcMondo: <http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/>
+prefix mondo: <http://purl.obolibrary.org/obo/mondo#>
+
+# Tests if an animal disease made it into the rare disease subset
+
+SELECT DISTINCT ?entity ?property ?value WHERE
+{
+  VALUES ?property { <http://purl.obolibrary.org/obo/mondo#rare> }
+  ?entity <http://www.geneontology.org/formats/oboInOwl#inSubset> ?property .
+  ?entity rdfs:subClassOf* <http://purl.obolibrary.org/obo/MONDO_0005583>
```

Comment:
Just providing my code reviewer face check. Syntax and everything looks OK to me. Verified that `MONDO_0005583` is for animal disease, and looks like correctly selects that which is in the rare disease subset.

I actually don't 100% know how filter this works:
`?entity mondo:excluded_from_qc_check mondoSparqlQcMondo:qc-animal-disease-rare.sparql .`
I didn't know we had this fine-grained exclusion of things from QC check. Looks like not only is there a predicate for excluding from QC check, but there is this weird file name as property value that I've never seen before. I'm going to guess perhaps that there are currently no such exclusions, but that this is a standard thing we put in our QC checks so that we can easily add exclusions in the future.

Only a minor cosmetic suggestion:
It looks like it returns distinct `?entity ?property ?value`: `CURIE <http://purl.obolibrary.org/obo/mondo#rare> "Animal disease in Rare subset"`
If I'm correct about that this change is more readable`?entity ?value`: `CURIE "Animal disease in Rare subset"`

Reply:
> I actually don't 100% know how filter this works:
?entity mondo:excluded_from_qc_check mondoSparqlQcMondo:qc-animal-disease-rare.sparql .

Yes, the main problem is that there are a number of very useful QC checks that do not _always_ apply, but most of the time. So we invented this mechanism to allow curators to explicitly bypass a QC check. `mondo:excluded_from_qc_check` is a relation in Mondo (annotation property) and `mondoSparqlQcMondo:qc-animal-disease-rare.sparql` is the PURL of the QC check called `animal-disease-rare`. 

If you look at mondo-edit.obo, you will find many such exclusions on other tests.

 > Only a minor cosmetic suggestion:

All QC checks return the `?entity ?property ?value` in that order, which is a prerequisite of using them in the context of `robot report`. Now it returns

```
http://purl.obolibrary.org/obo/MONDO_0025135,http://purl.obolibrary.org/obo/mondo#rare,Animal disease in Rare subset
```

Can you say again what you would think is better to bind to each of the three variables?

Reply:
Thanks for taking time to get me up to speed!

As `?entity ?property ?value` is the standard for all QC returns (I didn't know), then I suggest we change nothing.

All I was saying was that "mondo#rare" is implicit in "Animal disease in Rare subset".